Remotion/Validation/Core/MetaValidation/Rules/Custom/AnyRuleAppliedPropertyMetaValidationRule.cs

```diff
@@ -49,7 +49,7 @@ yield return
             MetaValidationRuleValidationResult.CreateInvalidResult (
                 "'{0}' failed for property '{1}.{2}': No validation rules defined.",
                 GetType ().Name,
-                _property.ReflectedType.GetFullNameSafe(),
+                _property.ReflectedType?.GetFullNameSafe(),
```

Comment:
I don't think that a TODO is necessary, if we ever decide to add an extension method, we'll check every occurrence anyway. I'll change the `?` to `!`.

Reply:
\+ I don't really think the nullability of ReflectedType should bother us if you think about when it would be `null`. From MSDN:

> If the MemberInfo object is a global member (that is, if it was obtained from the Module.GetMethods method, which returns global methods on a module), the returned DeclaringType will be null.

😉

Reply:
This refers to `DeclaringType`, but I guess it's the same for `ReflectedType`, right?